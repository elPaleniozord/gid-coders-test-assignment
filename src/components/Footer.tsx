const Footer = () => (
  <footer>
    Designed and coded by jWalasik
  </footer>
)

export default Footer