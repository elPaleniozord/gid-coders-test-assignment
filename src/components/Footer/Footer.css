.footer {
  width: 100%;
  height: 1.2rem;
  font-size: .8rem;
  background: rgb(32, 31, 31);
  color: white;
  position: fixed;
  bottom: 0;
}