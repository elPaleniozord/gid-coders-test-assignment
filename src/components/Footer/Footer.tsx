import './Footer.css'

const Footer = () => (
  <footer className='footer'>
    Designed and coded by <a className='footer-link' href='https://github.com/elPaleniozord'>jWalasik</a>
  </footer>
)

export default Footer