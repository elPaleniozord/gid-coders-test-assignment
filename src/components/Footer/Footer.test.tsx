import { render } from '@testing-library/react';
import Footer from "./Footer"

test('should render Header Component correctly', () => {
  const { container } = render(<Footer />)
  const footer = container.querySelector('footer')

  expect(footer).toMatchSnapshot()
}) 