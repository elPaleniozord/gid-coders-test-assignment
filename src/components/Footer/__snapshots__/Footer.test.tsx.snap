// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Header Component correctly 1`] = `
<footer
  class="footer"
>
  Designed and coded by 
  <a
    class="footer-link"
    href="https://github.com/elPaleniozord"
  >
    jWalasik
  </a>
</footer>
`;
